src/react-apps/ux-editor/src/containers/Container.tsx

```diff
@@ -0,0 +1,197 @@
+  id: string;
+  baseContainer?: boolean;
+}
+
+export interface IContainerProps extends IProvidedContainerProps {
+  dataModelGroup?: string;
+  itemOrder: any;
+  components: any;
+  containers: any;
+  repeating: boolean;
+  designMode: boolean;
+  formData: any;
+  index?: number;
+}
+
+export class ContainerComponent extends React.Component<IContainerProps> {
+  
+  public handleContainerDelete = (e: any) => {
+    FormDesignerActionDispatchers.deleteFormContainer(this.props.id);
+    e.stopPropagation();
+  }
+
+  public handleComponentDataUpdate = (
+    id: string,
+    dataModelElement: IDataModelFieldElement,
+    callbackValue: any,
+  ): void => {
+    if ((this.props.index || this.props.index > -1) && this.props.dataModelGroup && this.props.repeating) {
+      const dataBindingName = dataModelElement.DataBindingName.replace(this.props.dataModelGroup,
+        this.props.dataModelGroup + `[${this.props.index}]`);
+      FormFillerActionDispatchers.updateFormData(
+        id,
+        callbackValue,
+        dataModelElement,
+        dataBindingName,
+        false,
+      );
+    } else {
+      FormFillerActionDispatchers.updateFormData(
+        id,
+        callbackValue,
+        dataModelElement,
+        dataModelElement.DataBindingName,
+        true,
+      );
+    }
+
+    ConditionalRenderingActionDispatcher.checkIfConditionalRulesShouldRun();
+    ApiActionDispatchers.checkIfApiShouldFetch(id, dataModelElement, callbackValue);
+    RuleConnectionActionDispatchers.checkIfRuleShouldRun(id, dataModelElement, callbackValue);
+  }
+
+  public render() {
+    return (
+      <div>
+      <div
+        className={'col-12'}
+        style={this.props.baseContainer ? {} :
+          { border: '1px dashed #1eaef7', marginTop: '10 px', marginBottom: '10px'}}
+      >
+        <div className='col-1'>
+          {this.renderDeleteGroupButton()}
+        </div>
+        {this.props.itemOrder.map((id: string, index: number) => (
+          this.props.components[id] ? this.renderFormComponent(id, index) :
+            (this.props.containers[id] ? this.renderContainer(id) : null)
+        ))}
+      </div>
+      {this.renderNewGroupButton()}
+      </div>
+    );
+  }
+
+  public renderContainer = (id: string) => {
+    return (
+      <Container
+        id={id}
+        key={id}
+        baseContainer={false}
+      />
+    );
+  }
+
+  public renderDeleteGroupButton = (): JSX.Element => {
+    if (this.props.baseContainer) return null;
+    return (
+          <button
+            type='button'
+            className='a-btn a-btn-icon p-0'
+            onClick={this.handleContainerDelete}
+          >
+            <i className='ai ai-circle-exit a-danger ai-left' />
+          </button>
+    );
+  }
+
+  public renderNewGroupButton = (): JSX.Element => {
+    if (this.props.baseContainer || !this.props.repeating) return null;
+    const repeatingGroupCount = Object.keys(this.props.containers).filter((id) => {
```

Comment:
Unclear variabel-name. It's an array of booleans (if I'm not mistaken), not a number as a count would be.

Reply:
Updated so that it is actually a number (since the array is not used anywhere else)